c/apps/lf-workspace/hello/src/Makefile

```diff
@@ -0,0 +1,38 @@
+LF_PROJECT_ROOT := "/home/ehsan/t-crest/patmos/c/apps/lf-workspace/hello/"
```

Comment:
This path is hardcoded to Ehsan's machine, so will fail on any other. Should use relative paths